staging/src/k8s.io/api/apps/v1beta2/types.go

```diff
@@ -191,11 +191,11 @@ const (
 	// PersistentVolumeClaims associated with StatefulSet VolumeClaimTemplates
 	// will not be deleted.
 	RetainPersistentVolumeClaimRetentionPolicyType PersistentVolumeClaimRetentionPolicyType = "Retain"
-	// RetentionPersistentVolumeClaimRetentionPolicyType specifies that
+	// DeletePersistentVolumeClaimRetentionPolicyType specifies that
 	// PersistentVolumeClaims associated with StatefulSet VolumeClaimTemplates
 	// will be deleted in the scenario specified in
 	// StatefulSetPersistentVolumeClaimRetentionPolicy.
-	RetentionPersistentVolumeClaimRetentionPolicyType PersistentVolumeClaimRetentionPolicyType = "Delete"
+	DeletePersistentVolumeClaimRetentionPolicyType PersistentVolumeClaimRetentionPolicyType = "Delete"
```

Comment:
@Jefftree would renaming the field here be breaking for v1.32 users using --emulated-version=v1.31 with old configs using the beta feature gate?  Does storage versions account for this in any way?

Reply:
ah, this was an old typo that I fixed. The type isn't used anywhere in k/k, but if you want me to revert it in case it's being used elsewhere I can.

Reply:
yeah this field doesn't look to be used with the feature so it should be more of an API review question than compatibility version one.

Reply:
I see thanks.  This change LGTM with respect to Compatibility Versions

Reply:
go-level changes have no bearing on the content of the REST API